tests/integration/src/server/http_server.py

```
@@ -195,7 +326,9 @@ def run_server(port=60000, *, use_tls=False, tls_crt_file=None, tls_key_file=Non
     if use_tls:
         ssl_context = (tls_crt_file, tls_key_file)
 
-    server_instance = make_server("0.0.0.0", port, app, ssl_context=ssl_context)
+    server_instance = make_server("0.0.0.0", port, app,
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)